pom.xml

```diff
@@ -53,7 +53,6 @@
     <properties>
         <maven.compiler.target>11</maven.compiler.target>
         <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.release>11</maven.compiler.release>
```

Comment:
This is required to avoid build error:

```
exporting a package from system module java.base is not allowed with --release
```

Reply:
Hmmm. This seriously needs to be discussed.